test/direct_chunk.c

```diff
@@ -76,9 +76,9 @@
 
 /* Local prototypes for filter functions */
 static size_t filter_bogus1(unsigned int flags, size_t cd_nelmts, const unsigned int *cd_values,
-                            size_t nbytes, size_t *buf_size, void **buf);
+                            size_t nbytes, const size_t *buf_size, void **buf);
 static size_t filter_bogus2(unsigned int flags, size_t cd_nelmts, const unsigned int *cd_values,
-                            size_t nbytes, size_t *buf_size, void **buf);
+                            size_t nbytes, const size_t *buf_size, void **buf);
```

Comment:
These 2 may not work, since they are supposed to match the signature for H5Z_func_t